app/src/main/java/de/westnordost/streetcomplete/quests/SplitWayFragment.kt

```diff
@@ -76,7 +76,7 @@ class SplitWayFragment : Fragment(R.layout.fragment_split_way),
         osmQuestKey = Json.decodeFromString(args.getString(ARG_OSM_QUEST_KEY)!!)
         way = Json.decodeFromString(args.getString(ARG_WAY)!!)
         val elementGeometry: ElementPolylinesGeometry = Json.decodeFromString(args.getString(ARG_ELEMENT_GEOMETRY)!!)
-        positions = elementGeometry.polylines.single().map { LatLon(it.latitude, it.longitude) }
+        positions = elementGeometry.polylines.single()
```

Comment:
There's not quite as many usages of `Location.toLatLon()` as I though there would be, because it turns out the list we're mapping over here is already a `List<LatLon>`. So I removed the redundant(?) `map`.

I'm not sure if there's a reason why we want a fresh set of objects here, so if there is, I'll revert this part.

Reply:
`LatLon` is immutable, so passing it around freely is not a problem. I think there was a don't remember why there was a `.map`, it doesn't make sense (now)